inotify.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procfs
+
+import (
+	"bufio"
+	"os"
+	"regexp"
+	"strings"
+)
+
+// InotifyStat represents a file descriptor's inotify watch count.
+type InotifyStat struct {
+	// File descriptor
+	FD string
+	// List of inotify lines in the fdinfo file
+	Lines []string
```

Comment:
Instead of just loading the line here, I think it would be better to parse the values into their expected fields/types.  Or at least add the values to map for easy lookup.